### PR TITLE
add schema definitions

### PR DESCRIPTION
**Pull request by olaurino**
This PR adds the current VOTable schema(ta) to the document (and the repository).

I know the discussion on the schema is still on, but it's probably even a better reason for including it now and then record the changes as they happen while we discuss the schemata.

Since the PR has not been reviewed yet, I am using it to include the actual documentation, too.

### CODE REVIEW

**Comment by olaurino** [comment]
Rebasing to resolve conflicts, original tip was cf5977b

File: 002-mapping-vodml.md [hunk 1]
```diff
@@ -62,11 +63,11 @@ be instances of structured types.
 
 The extension to the VOTable schema is reproduced in [@sec:schema].
 
-VODMLReference
+VODMLReference { #sec:vodmlref }
 --------------
 
 This XML type represents a reference to a single element in a VO-DML/XML
-document. It take over the role of the `@utype` attribute in this regards.
+document. It takea over the role of the `@utype` attribute in this regards.
```
Comment (glemson) [inline 002-mapping-vodml.md:70]: type: takea -> takes

File: 002-mapping-vodml.md [hunk 2]
```diff
@@ -185,10 +186,10 @@ Broadly speaking, this specification is all about Object-Relational
 Mapping (ORM). Data Models are represented in VO-DML according to an
 Entity-Relationship paradigm, in a fashion that is implementable
 by relational databases, object oriented languages, and possibly
-to certain document oriented dabases as well.
+by certain document oriented data bases as well.
 
 As VOTable can represent several tables in the same file with rich
-metadata, one can look at VOTable as a database that can represent
+metadata, one can look at VOTable as a data base that can store instances of
```
Comment (glemson) [inline 002-mapping-vodml.md:192]: why space in 'data base'?

**Review by glemson — APPROVED** [review]
I will comment on contents once everything is assembled.

**Pull request merged**